test/methods.js

```diff
@@ -396,6 +396,29 @@ asyncTest( "remote", function() {
 	strictEqual( v.element( e ), true, "still invalid, because remote validation must block until it returns; dependency-mismatch considered as valid though" );
 } );
 
+asyncTest( "remote, pending class added to element while call outstanding", function() {
+	expect( 3 );
+	var e = $( "#username" ),
+		v = $( "#userForm" ).validate( {
+			rules: {
+				username: {
+					remote: {
+						url: "users.php",
+						complete: function() {
+							strictEqual( e.hasClass( "pending" ), false, "not pending since ajax call complete" );
+							start();
+						}
+					}
+				}
+			}
+		} );
+	strictEqual( e.hasClass("pending"), false, "not pending since no data entered" );
```

Comment:
Please add one space:
```js
                        ↓       ↓
strictEqual( e.hasClass("pending"), false, "not pending since no data entered" );
```